pkg/usageStats/stats.go

```diff
@@ -123,12 +125,28 @@ func ReadQueryStats(orgid uint64) error {
 		return err
 	}
 	if len(val) > 0 {
-		flushedQueriesSinceInstall, err := strconv.ParseUint(val[len(val)-1][0], 10, 64)
+		lastRecord := val[len(val)-1]
+		if len(lastRecord) < 1 {
+			return fmt.Errorf("readQueryStats: last record has insufficient fields %v", lastRecord)
+		}
```

Comment:
You should also log this error, just in case the caller of this function doesn't log the error. You can do it like this:
```
return utils.TeeErrorf("readQueryStats: last record has insufficient fields %v", lastRecord)
```

Same comment some other places in this function